src/app/shared/components/template/components/task-card/task-card.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit } from "@angular/core";
+import { TaskService } from "src/app/shared/services/task/task.service";
+import { getStringParamFromTemplateRow } from "src/app/shared/utils";
+import { TemplateBaseComponent } from "../base";
+import { IProgressStatus } from "src/app/shared/services/task/task.service";
+
+@Component({
+  selector: "plh-task-card",
+  templateUrl: "./task-card.component.html",
+  styleUrls: ["./task-card.component.scss"],
+})
+export class TmplTaskCardComponent extends TemplateBaseComponent implements OnInit {
+  style: string;
+  displayType: "landscape" | "portrait";
+  highlighted: boolean = true;
+  highlightedText = "Active";
+  progressStatus: IProgressStatus = "notStarted";
+  taskGroupId: string | null;
+  taskId: string | null;
+  title: string | null;
+  subtitle: string | null;
+  image: string | null;
+  completedIcon: string;
+  inProgressIcon: string;
+
+  constructor(private taskService: TaskService) {
+    super();
+  }
+
+  ngOnInit() {
+    this.getParams();
+    this.highlighted = this.taskService.checkHighlightedTask(this.taskId || this.taskGroupId);
+  }
+
+  getParams() {
+    this.taskGroupId = getStringParamFromTemplateRow(this._row, "task_group_id", null);
+    this.taskId = getStringParamFromTemplateRow(this._row, "task_id", null);
+    this.title = getStringParamFromTemplateRow(this._row, "title", null);
+    this.subtitle = getStringParamFromTemplateRow(this._row, "subtitle", null);
+    this.image = getStringParamFromTemplateRow(this._row, "image", null);
+    this.style = getStringParamFromTemplateRow(this._row, "style", "landscape");
```

Comment:
nit(non-blocking)
What's the difference between displayType and style? If styles are going to be passed by authors we really should make the list of accepted styles explict so that they are easier to document